config/locales/en.yml

```diff
@@ -163,6 +163,7 @@ en:
   contact_info: Contact
   social_info: Social
   edit_user: Edit User
+  delete_user: Delete User
```

Comment:
we should add this line to all translation files but leave it commented, as this indicates it needs to be translated